bedrock/security/templates/security/bug-bounty/faq.html

```diff
@@ -15,15 +15,15 @@ <h1 class="mzp-c-article-title">Bug Bounty Program FAQ</h1>
   </header>
 
   <p>This FAQ attempts to answer various questions about the Mozilla
-    security bug bounty program sponsored by the Mozilla Foundation. For
+    security bug bounty program sponsored by Mozilla Foundation. For
     more information see the
     <a href="{{ url('security.bug-bounty') }}">official guidelines</a>
     governing the program.</p>
 
   <h3>General questions</h3>
 
   <ul class="mzp-u-list-styled">
-    <li><a href="#why">Why is the Mozilla Foundation doing this?</a></li>
```

Comment:
@fkiriakos07 Just in terms of "module ownership" do you have any issue with this wording change here? Our team expects not, but just being careful

Reply:
thanks for checking in, I don't think there is any issue here.